cookiecutter.json

```diff
@@ -33,6 +33,17 @@
     "GCP",
     "None"
   ],
+  "mail_service": [
+    "{% if cookiecutter.cloud_provider == 'AWS' %}Amazon SES{% else %}Plain/Vanilla Django-Anymail{% endif %}",
+    "Mailgun",
+    "Mailjet",
+    "Mandrill",
+    "Postmark",
+    "Sendgrid",
+    "SendinBlue",
+    "SparkPost",
+    "Plain/Vanilla Django-Anymail"
```

Comment:
As a user of the template, I'm not sure what "plain/vanilla" implies while it's pretty clear that the others are provider name. Is it going to spin up a local for me and send emails directly from Django? Do I need to set this up myself?

Since the question is for "Mail service", maybe we could label this option as "Other (SMTP)"? I'm open to suggestions if you think of a better name.